bin/main.html

```diff
@@ -120,7 +120,7 @@ <h2>Who do you want to be today?</h2>
           <label class="action-area-text" for="attr_roll_advance_boost">Boost</label>
           <input class="narrowish" id="attr_roll_advance_boost" name="attr_roll_advance_boost" type="number" value="0" />
           <label class="action-area-text" for="attr_action_roll_EP">PWR</label>
-          <input class="narrow" id="attr_action_roll_EP" name="attr_action_roll_EP" value="@{roll_advance_boost}" disabled="true" />
+          <input class="narrow" id="attr_action_roll_EP" name="attr_action_roll_EP" value="@{roll_advance_boost} + @{action_feats} + @{action_EP}" disabled="true" />
```

Comment:
I think this is the better approach than before.
Sum up the three feats so that you can compare it to your available PWR.
![image](https://github.com/user-attachments/assets/c91bcbcf-3dfd-450e-9e3d-de7077f5f614)